scripts/initialize.sh

```diff
@@ -72,9 +76,46 @@ while getopts "n:o:p:d:b:a:s:k:h" opt; do
 done
 shift $(expr $OPTIND - 1) # remove options from positional parameters
 
+CONFIG_DIR=${config_dir:-$CONFIG_DIR_DEFAULT}
+
+# read from the config file if a stake_owner is not provided as parameter
 if [ -z "$stake_owner" ]; then
-   echo 'Stake owner address must be provided. See --help'
-   exit 1
+   printf "\nReading stake owner address from the config file..."
+
+   # read from the config file when the stake owner is not provided
+   config_files=($CONFIG_DIR/*.toml)
+   config_files_count=${#config_files[@]}
+   while :; do
+      printf "\nSelect client config file: \n"
+      i=1
+      for o in "${config_files[@]}"; do
+         echo "$i) ${o##*/}"
+         let i++
+      done
+
+      read reply
+      if [ "$reply" -ge 1 ] && [ "$reply" -le $config_files_count ]; then
+         CONFIG_FILE_PATH=${config_files["$reply" - 1]}
+         break
+      else
+         printf "\nInvalid choice. Please choose an existing option number.\n"
+      fi
+   done
+
+   sed -n -e '/^keyFile -/p' $CONFIG_FILE_PATH
+
+   key_file_str=$(grep "^keyFile" $CONFIG_FILE_PATH)
+   # internal field separator, creates array from key_file separated by '"' sign
+   # array[1] is the key file path
+   IFS='"' read -r -a array <<<"$key_file_str"
```

Comment:
We could try using `awk`:
```suggestion
   key_file_path=$(awk '/keyFile ?=(.*)$/{print $3}' $CONFIG_FILE_PATH)
```

Reply:
We need to strip the quotes, so:
```suggestion
   key_file_path=$(awk '/keyFile ?=(.*)$/{print $3}' $CONFIG_FILE_PATH | xargs)
```

Reply:
[1265fa5](https://github.com/keep-network/keep-core/pull/3163/commits/1265fa59d426363fee21ebfe1188d5e73bdd43be)